main.go

```diff
@@ -117,18 +125,47 @@ func main() { //nolint:funlen
 
 	ctrl.SetLogger(logger.ConfigLoggers(logmode))
 
+	//	var namespaceSelector labels.Set{labels.ODH.OwnedNamespace: "true"}.AsSelector()
+	// CSVFields := fields.Set{"spec.displayName": string(cluster.OpenDataHub)}
```

Comment:
```suggestion
```

Do we need to keep this commented out code? What's the value?

Reply:
removed. was left from previous trying to cache namespace and CSV